types/emscripten/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6",
+            "es2020",
```

Comment:
What about just adding "es2020" instead of removing "es6"?

I don't think HEAP64/HEAPU64 requires removing "es6" flag though.

Reply:
Yup, fixed